stubs/urllib3/urllib3/filepost.pyi

```diff
@@ -1,12 +1,15 @@
-from typing import Any
+from typing import Any, Iterable, Mapping, Sequence
 
 from . import fields
 
 RequestField = fields.RequestField
 
 writer: Any
 
-def choose_boundary(): ...
-def iter_field_objects(fields): ...
+_TYPE_FIELDS_SEQUENCE = Sequence[tuple[str, fields._TYPE_FIELD_VALUE_TUPLE] | RequestField]
```

Comment:
`fields._TYPE_FIELD_VALUE_TUPLE` exists on the `main` branch of `urllib3`, but doesn't exist in our stubs for `fields.pyi` yet (nor in any released version of `urllib3`). Feel free to add it as part of this PR though!